standards/views.py

```diff
@@ -37,6 +38,26 @@ def by_curriculum(request, slug):
     return render(request, 'standards/curriculum_nogrid.html', {'curriculum': curriculum})
 
 
+def by_curriculum_csv(request, slug):
+    curriculum = get_object_or_404(Curriculum, slug=slug)
+
+    response = HttpResponse(content_type='text/csv')
+    response['Content-Disposition'] = 'attachment; filename="%s_standards.csv"' % curriculum.slug
+
+    writer = csv.writer(response)
+    writer.writerow(['curriculum', 'unit', 'lesson #', 'lesson name',
+                     'standard framework', 'standard', 'cross curricular opportunity'])
+    for unit in curriculum.units:
+        for lesson in unit.lessons:
+            for standard in lesson.standards.all():
+                writer.writerow([curriculum.slug, unit.slug, 'lesson %d' % lesson.number, lesson.title,
+                                 standard.framework.slug, standard.shortcode, ''])
+            for standard in lesson.opportunity_standards.all():
+                writer.writerow([curriculum.slug, unit.slug, 'lesson %d' % lesson.number, lesson.title,
+                                 standard.framework.slug, standard.shortcode, 'X'])
```

Comment:
nit: is there a specific reason to use `''`/`'X'` rather than `true`/`false`?

Reply:
switched